Translations/it.lproj/InfoPlist.strings

```diff
@@ -57,7 +57,7 @@
 "Safari extension" = "Estensione di Safari";
 "ACE Archive" = "Archivio ACE";
 "XIP Secure Archive" = "Archivio sicuro XIP";
-"Electronic Book" = "Electronic Book";
+"Electronic Book" = "Pubblicazione elettronica";
 "Apple Archive" = "Apple Archive";
```

Comment:
This one is translated in the bundled macOS extractor tool so might as well translate it 😛